roles/apim-gateway/templates/carbon-home/repository/conf/api-manager.xml.j2

```diff
@@ -16,11 +16,11 @@
          logic relies on this. -->
     <AuthManager>
         <!-- Server URL of the Authentication service -->
-        <ServerURL>https://localhost:${mgt.transport.https.port}${carbon.context}services/</ServerURL>
+        <ServerURL>https://{{ key_manager_hostname }}:{{ key_manager_port }}${carbon.context}services/</ServerURL>
```

Comment:
Isn't it good to use `auth_manager` rather than using `key_manager` here? Any specific reason?

Reply:
<AuthManager> should be given the hostname of the APIM key_manager in order to create the server URL of the Authentication service. Hence the name "key_manager_hostname"